cluster/ci/config/prow/config.yaml

```diff
@@ -4427,6 +4428,61 @@ postsubmits:
         - --target=[images]
         - --promote
 
+  openshift/machine-api-operator:
+  - name: branch-ci-openshift-machine-api-operator-images
+    agent: kubernetes
+    labels:
+      artifacts: images
+    branches:
+    - master
+    decorate: true
+    skip_cloning: true
+    spec:
+      serviceAccountName: ci-operator
+      containers:
+      - name: test
+        image: ci-operator:latest
+        env:
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              name: ci-operator-openshift-machine-api-operator
+              key: master.json
+        command:
+        - ci-operator
+        args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --target=[images]
+        - --promote
+
+  - name: pull-ci-openshift-machine-api-operator-unit
```

Comment:
This needs to be under "presubmits" (with other PR jobs) and the branch job should be under "postsubmits".  The section you are in determines when you are called.

Reply:
Moved